warehouse/templates/pages/help.html

```diff
@@ -281,23 +281,24 @@ <h3 id="availability">{{ availability() }}</h3>
           PyPI is heavily cached and distributed via CDN thanks to our sponsor <a href="https://www.fastly.com/">Fastly</a> and thus is generally available globally. However, as could be expected for a giant distributed system, things can and sometimes do go wrong. If you have high availability requirements for your package index, consider either a <a href="#mirroring">mirror</a> or a <a href="#private-indices">private index</a>.
         </p>
 
-        <h3 id="preproduction">{{ preproduction() }}</h3>
+        <h3 id="beta">{{ beta() }}</h3>
         <p>
-          'Pre-production' indicates that you are using a version of this software that is not fully tested and 'production ready'. This means:
+          'Beta' indicates you are using a version of PyPI that is robust, but not <i>fully</i> tested and 'production ready'. This means:
         </p>
         <ul>
           <li>The UI may return unusual or erroneous results</li>
           <li>Performance may not yet be optimised</li>
-          <li>Some features that you use on the old interface may be missing</li>
+          <li>Some features that you used in <a href="https://pypi.python.org">the old interface (pypi.python.org)</a> may be missing</li>
+	  <li>We <a href="#feedback">need your feedback</a> to get the site production ready</li>
         </ul>
         <p>
-          While we are in pre-production, our infrastructure cannot yet support all of the traffic generated by our users running <code>pip install</code>. We therefore ask that you do not use this version of PyPI for production codebases.
+          While we are in beta, the pypi.org infrastructure cannot yet support all of the API traffic generated by our users running <code>pip install</code>, so don't explicitly point to it in your production setups. But we're occasionally redirecting portions of that traffic in load tests we announce <a href="http://status.python.org/">on our status page</a>.
```

Comment:
I think we can say we're generally ready for traffic? Or will be once @ewdurbin gets the XMLRPC caching sorted out? Are we ready to tell people they can start shifting their own traffic to Warehouse now? I'm not sure how to word it really, but like "Early Adopter" traffic is probably not going to knock us over?

Reply:
I'd like to encourage early adopter traffic, and can find a way to word it if @ewdurbin says it's ok.

Reply:
Once we get #3336 cleared up, we'll be all set to start calling it beta. Right now it's just too costly if a user with lots of projects logs in.

Reply:
@ewdurbin Good info, thanks. So, once we're in beta, what level of traffic are you ok with encouraging? (this is kind of "what does 'beta' mean to you?")

Reply:
Anything that arrives organically is probably fine. We may be migrating our backend database as well now... so perhaps we should wait until after that?